recipes/hazelcastcxx/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class HazelcastCxx(ConanFile):
+    name = "hazelcastcxx"
+    description = "Hazelcast is C++ API for in memory database."
+    license = "Apache 2.0"
+    topics = ("conan", "hazelcast", "client", "database", "cache")
+    homepage = "https://github.com/hazelcast/hazelcast-cpp-client"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "static": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "static": True,
+        "fPIC": True,
+        "with_openssl": False
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def requirements(self):
+        self.requires("boost/1.75.0")
```

Comment:
I guess this one should be `boost/1.71.0` ?

Reply:
This does not mean minimum. It uses the exact version.

Reply:
We like to use the exact version to make sure the build are reproducible